TLM/TMPE.API/Traffic/Data/SpeedValue.cs

```diff
@@ -0,0 +1,156 @@
+namespace TrafficManager.API.Traffic.Data {
+    using System;
+    using JetBrains.Annotations;
+    using UnityEngine;
+
+    /// <summary>
+    /// Represents a speed value expressed in km/hour.
+    /// </summary>
+    [Serializable]
+    public struct KmphValue {
+        public KmphValue(ushort kmph) {
+            Kmph = kmph;
+        }
+
+        public override string ToString() {
+            return $"{Kmph:0.0} km/h";
+        }
+
+        public ushort Kmph { get; set; }
```

Comment:
Mutable structs are evil...

Reply:
It is still just a single wrapped float. How bad can copying a float be?

Reply:
https://stackoverflow.com/a/441357/5240636

Reply:
Your struct is mutable. It has a writable property. This is a bad design in most cases an should be avoided. It can lead to unexpected behavior and obscure bugs. Search SO for details.
Better make your struct `readonly` and your property `get`-only. 
Note: the structs in the game (like `Citizen`) are mutable, but this is the needed pay-off for the performance: they are used as data storage inside of a continuous entity array. Your struct is not an entity, so there is no need to keep it mutable.

Reply:
That thing should be forbidden on the language level :P I'll get back to you when this is read-only.

Reply:
Read-only now. I may be noob in C# but i know a tiny bit about virtual machines :)